talentsearch/resources/js/talentSearchRoutes.ts

```diff
@@ -0,0 +1,2 @@
+export const homePath = (): string => "/";
+export const searchDashboardPath = (): string => `${homePath()}search`;
```

Comment:
```suggestion
import { BASE_URL } from "./talentSearchConstants";

export const homePath = (): string => `/${BASE_URL}`;
export const searchDashboardPath = (): string => `${homePath()}/search`;
```

Reply:
I don't think `/${BASE_URL}`, but I've pushed up some changes that hopefully resolve what you but looking for in this file as well as _talentSearchRoutes.ts_.